test/e2e/providers_test.go

```diff
@@ -60,6 +61,26 @@ func TestProvidersSetStatuspageID(t *testing.T) {
 	}
 }
 
+func TestProvidersSetStatuspageIDFlag(t *testing.T) {
```

Comment:
Might be good to rename the file to `providers_statuspage_test.go`.  I bet we will get more providers.

Reply:
Only thing I see, otherwise lgtm!

Reply:
This can be done later, is not a big deal.  Lets just make sure we don't end up with a future bloated test file!